frontend/src/components/views/RangeIndicatorView.jsx

```diff
@@ -0,0 +1,37 @@
+import { pythagorean, round } from '../../utils/math_utils';
+import { convertToRenderCoordinates } from '../../utils/render_utils';
+
+export const RangeIndicatorView = ({ tower, gameMap }) => {
+    const { minRange, maxRange } = tower;
+    const SQUARE_SIZE = 1/3;
+
+    let tiles = [];
+    for (let x = -SQUARE_SIZE; x < gameMap.width-SQUARE_SIZE; x += SQUARE_SIZE) {
+        for (let y = -SQUARE_SIZE; y < gameMap.depth-SQUARE_SIZE; y += SQUARE_SIZE) {
```

Comment:
(0,0) isn't the top left edge of the first tile, it's actually the center. (The spawner portal is at 0,0) 
Because we're splitting each tile into nine squares, the center is (M,N), and the "first" square is (M-1/3,N-1/3)

Reply:
Oh, I misunderstood, got it